### PR TITLE
Migrate Add_a_column1 from devteam

### PR DESCRIPTION
**Pull request by mvdbeek**
First commit is just a copy from devteam's master branch.

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by mvdbeek** [comment]
Passed on my fork: https://travis-ci.org/mvdbeek/tools-iuc/jobs/549772189

File: tools/column_maker/column_maker.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    (sed -n '1,1p' '$input' | sed  "s|$|%${header_lines_conditional.header_new_column_name}|" | tr "%" "\t") > header &&
+    sed '1,1d' '$input' > data &&
+#else:
+    touch header &&
+    ln -s '$input' data &&
+#end if
+
+python '$__tool_directory__/column_maker.py'
+    data column_maker_output
+    "$cond"
```
Comment (abretaud) [inline tools/column_maker/column_maker.xml]: single quote here?

Comment (mvdbeek) [inline tools/column_maker/column_maker.xml]:
```suggestion
    '$cond'
```

**Comment by bgruening** [comment]
Thanks! @abretaud please feel free to merge!

**Pull request merged**